requirements-devel.txt

```diff
@@ -84,6 +84,7 @@ oauthlib==3.2.2
 overrides==7.7.0
 packaging==24.1
 PasteDeploy==3.1.0
+patchelf==0.17.2.1
```

Comment:
As far as I know `requirements*.txt` files are auto-generated and we keep dependencies in `pyproject.toml` or `setup.py`. 
If I'm not mistaken this dependency should be added here, and then you should run `tools/freeze-requirements.sh` script.
https://github.com/canonical/snapcraft/blob/main/setup.py#L61